source/vscode/src/changelog.ts

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import * as vscode from "vscode";
+
+// The latest version for which we want to show the changelog page
+const CHANGELOG_VERSION = undefined; // <-- Update this when you want to show a new changelog to users
+
+export function registerChangelogCommand(
+  context: vscode.ExtensionContext,
+): vscode.Disposable[] {
+  return [
+    vscode.commands.registerCommand("qsharp-vscode.showChangelog", async () => {
+      const changelogUri = vscode.Uri.joinPath(
+        context.extensionUri,
+        "CHANGELOG.md",
+      );
+      await vscode.commands.executeCommand(
+        "markdown.showPreview",
+        changelogUri,
+        vscode.ViewColumn.One,
+        { locked: true },
```

Comment:
I can't find docs for this API/argument. What does `{locked: true}` do and why do we need it here?